agent/gather.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+ip=$(<node0-ip)
+
+ssh -o 'ConnectTimeout=30' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' core@${ip} << EOF
+# check is agent-gather command is present
+if ! command -v /usr/local/bin/agent-gather &> /dev/null
+then
+    echo "Skipping gathering agent logs, agent-gather script not present."
+    exit 0
+fi
+
+sudo /usr/local/bin/agent-gather -O | tar -xJvf -
```

Comment:
Why are we extracting the files here instead of just providing the tarball as a CI artifact, as I believe we do with the must-gather?

Reply:
That's just an oversight.